js/script.js

```diff
@@ -0,0 +1,88 @@
+window.onload = () => {
+  const guestSpeakers = [
+    {
+      image: './assets/images/toto.png',
+      name: 'Torger Christian Wolff',
+      role: 'Formula 1 manager',
+      bio: 'Shareholder and director in Formula 1',
+    },
+    {
+      image: './assets/images/checop.png',
+      name: 'Sergio Michel Pérez Mendoza',
+      role: 'f1 driver',
+      bio: 'Redbull Racing Driver',
+    },
+    {
+      image: './assets/images/lewish.png',
+      name: 'Lewis Carl Davidson Larbalestier Hamilton',
+      role: 'f1 driver',
+      bio: 'Mercedes AMG Petronas Motorsport',
+    },
+    {
+      image: './assets/images/christianh.png',
+      name: 'Christian Horner',
+      role: 'Formula 1 manager',
+      bio: 'Shareholder and director in Formula 1',
+    },
+    {
+      image: './assets/images/zacb.png',
+      name: 'Zak Brown',
+      role: 'Formula 1 manager',
+      bio: 'Shareholder and director in Formula 1',
+    },
+    {
+      image: './assets/images/binnoto.png',
+      name: 'Mattia Binotto',
+      role: 'Formula 1 manager',
+      bio: 'Shareholder and director in Formula 1',
+    },
+  ];
+/* eslint-disable */
+  function displaySpeakerInfo(speakerInfo) {
+    const speakers = `<div class="speaker-1">
+    <div class="speaker-image">
+        <img src=${speakerInfo.image} alt="speaker-1">
+    </div>
+    <div>
+        <h2>${speakerInfo.name}</h2>
+        <h3>${speakerInfo.role}</h3>
+        <h4>${speakerInfo.bio}</h4>
+    </div>
+</div>`;
+    return speakers;
+  }
+  const guestSpeakerAtEvent = document.querySelector('.image-display-1');
+  guestSpeakers.forEach((card) => {
+    guestSpeakerAtEvent.innerHTML += displaySpeakerInfo(card);
+  });
+
+  const hamburger = document.querySelector('.menu');
+  const closeBtn = document.querySelector('.closeBtn');
+
+  hamburger.addEventListener('click', () => {
```

Comment:
- [ ]Your hamburger shows pages, however we can only click on the about page; the other pages do not display and cannot take the user to the page he clicked. :wink:

Reply:
Hello and thank you very much for the comments. I will implement the changes, but in this section, the project only requires that function home and about @Alfredbis29

Reply:
thank you very much I thought it better and if you're right I gave it better I gave functionality to the other buttons